pkg/platform/aks/aks.go

```diff
@@ -292,7 +292,16 @@ func (c *config) Apply(ex *terraform.Executor) error {
 		return err
 	}
 
-	return ex.Apply()
+	return ex.Apply(nil)
+}
+
+// ApplyWithoutParallel applies Terraform configuration without parallel execution.
+func (c *config) ApplyWithoutParallel(ex *terraform.Executor) error {
+	if err := c.Initialize(ex); err != nil {
+		return err
```

Comment:
nit: all errors should be wrapped for traceability

Reply:
Fair, I'll leave other existing functions for now.